content/site.json

```diff
@@ -106,20 +106,28 @@
           "isLive": true
         },
         {
-          "name": "Under Construction: ACCURATE",
-          "link": "https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/how-to-participate/org-details/999999999/project/101138269/program/43108390/details",
-          "description": "Gaia-X Lighthouse ACCURATE aims to boost the competitiveness of European manufacturing companies with DSS and Manufacturing as a Service frameworks. ",
-          "image": "accurate-logo.webp",
-          "category": "manufacturing",
-          "isLive": false
```

Comment:
question(blocking): it looks like we have removed ACCURATE here - my guess is, this was by accident? Should we proceed with merging?